Blockzilla/OverlayView.swift

```diff
@@ -136,14 +139,18 @@ class OverlayView: UIView {
     
     fileprivate func updateCopyConstraint() {
         copyButton.snp.remakeConstraints { make in
-            if searchButton.isHidden || searchQuery.isEmpty {
+            if UIPasteboard.general.string != nil {
```

Comment:
If there are `if/else` statements that depend on it. You can create `bools` outside of the closure like 
`let searchActive = true` in that last if statement 
then in the closure you can do `if searchActive { make.top.equalTo(searchBorder) } `